xml/System.Diagnostics/Activity.xml

```diff
@@ -620,7 +639,16 @@ An ID has a hierarchical structure: `root-id.id1_id2.id3_`. The ID is generated
       <Docs>
         <summary>Gets the format for the <see cref="P:System.Diagnostics.Activity.Id" />.</summary>
         <value>The format for the <see cref="P:System.Diagnostics.Activity.Id" />.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
+
+## Remarks
+
+The best practice for the library authors is not try to set this property. This property is automatically set when creating the Activity. The setter is useful only in the case of Activity objects created using the Activity constructor and want to override the default value. Activity objects created using <xref:System.Diagnostics.ActivitySource.StartActivity> automatically have this property set to the right value.
```

Comment:
>best [](start = 4, length = 4)

This remark need to be moved to the IsAllDataRequested property and not here.

Reply:
Thanks. Fixed.